### PR TITLE
Updated Spanish translation and messages.pot file

### PR DESCRIPTION
**Pull request by dcapillae**
Updated messages.pot file to include [this change](https://github.com/internetarchive/openlibrary/pull/5272/files).

The Spanish translation has also been updated to reflect this change. I have taken the opportunity to correct some typos and poor translations.

### CODE REVIEW

**Comment by tfmorris** [comment]
That seems like a mechanical change. Couldn't all the translations be updated at the same time without any language knowledge being necessary?

**Comment by dcapillae** [comment]
@tfmorris I don't know what "mechanical change" means in this context. The mentioned change was a manual change made a few days.

The question about whether we can update all translations at the same time without any language knowledge being necessary is beyond my very limited knowledge.

**Comment by tfmorris** [comment]
@dcapillae I'm sorry, I should have been clearer. By "mechanical," I meant that the change appeared to be restricted to removing the HTML markup without changing the content or order of the text itself. On the surface, it seems like the same change could be applied to all languages, but perhaps I'm missing something.

**Comment by dcapillae** [comment]
Yes, I also think the HTML markup could be removed from all translations. It does not seem necessary to know the language in this case. It is a mechanical and very simple change. I did not do so in order to avoid interfering inappropriately with the work of other translators and to reduce possible conflicts with this PR, but I think it could be done if that's okay.

File: openlibrary/i18n/es/messages.po [hunk 1]
```diff
@@ -2512,12 +2264,8 @@ msgid "Choose a JPG, GIF or PNG on your computer,"
 msgstr "Elige un JPG, GIF o PNG en tu ordenador,"
 
 #: covers/add.html:69
-msgid ""
-"<span class=\"highlight\"><u>Or</u></span>, paste in the image URL if it's "
-"on the web."
-msgstr ""
-"<span class=\"highlight\"><u>O</u></span> pega la URL de la imagen si está "
-"en la web."
+msgid "<span class=\"highlight\"><u>Or</u></span>, paste in the image URL if it's on the web."
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:2267]: Yaaaay! Tests caught an actual error 🎉 

```suggestion
msgid "Or, paste in the image URL if it's on the web."
```

File: openlibrary/i18n/es/messages.po [hunk 2]
```diff
@@ -3175,8 +2857,8 @@ msgstr[1] "Esta edición está en <strong>%(count)s listas</strong>."
 #, python-format
 msgid "This work is on <strong>1 list</strong>."
 msgid_plural "This work is on <strong>%(count)s lists</strong>."
-msgstr[0] "Este trabajo está en <strong>1 lista</strong>."
-msgstr[1] "Este trabajo está en <strong>%(count)s listas</strong>."
+msgstr[0] "Esta obra está en <strong>1 lista</strong>."
+msgstr[1] "Esta obra está en <strong>%(count)s listas</strong>."
```
Comment (cdrini) [inline openlibrary/i18n/es/messages.po:2861]: :D

File: openlibrary/i18n/messages.pot [hunk 3]
```diff
@@ -2376,7 +2376,7 @@ msgid "Choose a JPG, GIF or PNG on your computer,"
 msgstr ""
 
 #: covers/add.html:69
-msgid "<span class=\"highlight\"><u>Or</u></span>, paste in the image URL if it's on the web."
+msgid "Or, paste in the image URL if it's on the web."
```
Comment (cdrini) [inline openlibrary/i18n/messages.pot:2379]: We'll likely need to do a ctrl-f in the other .po files, and fix this as well.

**Review by cdrini — APPROVED** [review]
Lgtm! One small fix I'll apply myself, and one fix we can do now (or in a future PR). Tossing on testing.openlibrary.org ...

**Comment by cdrini** [comment]
The scripts are a little heavy handed right now, because they rewrite all the files. We haven't run them recently. I'm planning on making them a little easier to diff, and then running them all in one go to regenerate everything in #5134

**Comment by cdrini** [comment]
I'm trusting the tests to have caught any potentially UI breaking changes ( like missing `%d` ). Thanks @dcapillae !

**Pull request merged**